_site/download.html

```diff
@@ -196,7 +196,7 @@ <h3 id="windows">
 
 
 <h3 id="macos">
-  <a class="header-anchor" href="#macos">macOS Big Sur / Monterey / Ventura / Sonoma</a>
+    <a class="header-anchor" href="#macos">macOS 12 (Monterey) / macOS 13 (Ventura) / macOS 14 (Sonoma) / macOS 15 (Sequoia)</a>
```

Comment:
This should be generated automatically rather than manually edited.
```suggestion
  <a class="header-anchor" href="#macos">macOS 12 (Monterey) / macOS 13 (Ventura) / macOS 14 (Sonoma) / macOS 15 (Sequoia)</a>
```